test/common/upstream/zone_aware_load_balancer_fuzz_base.cc

```diff
@@ -48,7 +48,6 @@ void ZoneAwareLoadBalancerFuzzBase::setupZoneAwareLoadBalancingSpecificLogic() {
   // Having 3 possible weights, 1, 2, and 3 to provide the state space at least some variation
   // in regards to weights, which do affect the load balancing algorithm. Cap the amount of
   // weights at 3 for simplicity's sake
-  stats_.max_host_weight_.set(3UL);
```

Comment:
here too.

Actually if the reason you woke to this is that (a) these stats were no longer accessible from this point and so coudn't be written and (b) that had no impact on the tests, then maybe send just the deletion of these max_host_weight_set() calls into a separate PR? That should be easy to review (deleting a bunch of superfluous test lines).

One concern I might have is if setting these values wakes up some code we need to be testable.

Reply:
removing the max_host_weight_ references in tests with PR: https://github.com/envoyproxy/envoy/pull/23936